client/src/Components/NavigationBar.js

```diff
@@ -0,0 +1,185 @@
+import React from "react";
+
+import { makeStyles } from "@material-ui/core/styles";
+import AppBar from "@material-ui/core/AppBar";
```

Comment:
As there are around 10 imports from the same library. You may want to consolidate the import statements into a single import statement for eg: import { Appbar, Toolbar } from "@material-ui/core"

Reply:
👍